packages/renderer/server/render-video.ts

```diff
@@ -60,18 +63,25 @@ export interface RenderVideoSettings {
  */
 async function initBrowserAndServer(
   fixedPort: number,
-  resolvedConfigPath: string,
+  projectFile: string,
+  outputFolderName: string,
   settings: RenderVideoSettings,
   params?: Record<string, unknown>,
 ) {
+  const resolvedProjectPath = path.join(process.cwd(), projectFile);
   const [browser, server] = await Promise.all([
     puppeteer.launch({headless: true, ...settings.puppeteer}),
     createServer({
-      configFile: resolvedConfigPath,
+      configFile: false,
       server: {
         port: fixedPort,
+        hmr: false,
+        host: settings.viteServerHost,
```

Comment:
We could expose the entire server objects in the settings by adding vite to `RenderVIdeoSettings` the same way we added `puppeteer?: BrowserLaunchArgumentOptions`.